src/Compilers/VisualBasic/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.vb

```diff
@@ -1653,6 +1653,21 @@ End Class
     End Sub
 
     <Fact>
+    Public Sub PublicSign_NoKey()
```

Comment:
Right -- I meant to turn this into a question.

Currently, for delay sign, signing without a key is a *warning*. That seems crazy. I would like to make this an error for public sign. I'm not sure how this could possibly be a warning for public sign, but I wanted to confirm that no one has any objection.